tsconfig.json

```diff
@@ -6,7 +6,9 @@
     "rootDir": "./src",
     "strict": true,
     "noImplicitAny": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "sourceMap": true,
```

Comment:
Consider whether 'sourceMap' and 'inlineSourceMap' are both necessary. 'sourceMap' generates separate .map files, while 'inlineSourceMap' embeds the source map in the output files. Having both might lead to confusion or unnecessary overhead.